typescript/lib/config.ts

```diff
@@ -302,6 +304,8 @@ export class AIConfigRuntime implements AIConfig {
     }
 
     const resolvedPrompt = modelParser.deserialize(prompt, this, params);
+    const endEvent = new CallbackEvent("on_resolve_end", __filename, {'result': resolvedPrompt});
```

Comment:
nit - don't need the `' '` around result. Also, could probably just do `{resolvedPrompt}` as the key/val